sdk/graphrbac/ci.yml

```diff
@@ -24,7 +24,11 @@ resources:
       name: Azure/azure-sdk-tools
       endpoint: azure
 
-jobs:
```

Comment:
I assume most of these changes are going to conflict with @mitchdenny's PR's to enable the unified pipeline for them.

Reply:
Well, I think it'll be easy to re-base.